sentry-android/src/main/java/io/sentry/android/AndroidSentryClientFactory.java

```diff
@@ -74,6 +79,28 @@ public SentryClient createSentryClient(Dsn dsn) {
     }
 
     @Override
+    protected Collection<String> getInAppFrames(Dsn dsn) {
```

Comment:
Ya, the problem is it's overriding one that already exists on `DefaultSentryClientFactory` (the name comes from a pre-1.0 config option name I think). 😢